docs/rule/no-invalid-role.md

```diff
@@ -32,6 +46,12 @@ This rule **allows** the following:
 <span role="none"></span>
 ```
 
+## Configuration
+
+* boolean - `true` to enable / `false` to disable
+
+* object - { 'catchNonexistentRoles' : `true|false` } -- Whether invalid role values should be allowed (defaults to `false`)
```

Comment:
Can you add a notice at the end of this sentence: `(TODO: change default to true in next major release)` to help us remember?